package.json

```diff
@@ -24,7 +24,7 @@
     "prepare": "is-ci || husky install",
     "pnpm:devPreinstall": "is-ci || node ./scripts/doctor.js",
     "test:example": "pnpm --filter \"example-*\" build",
-    "test:unit": "pnpm --filter \"@rspack/*\" test",
+    "test:unit": "NO_COLOR=1 pnpm --filter \"@rspack/*\" test",
```

Comment:
You can use `stats.colors = false` to omit the color

Reply:
Can not control swc's internal error emitter

Reply:
Ok, sounds that is going to make this PR not focused, it's move on to the next one for that.

Reply:
Seems https://github.com/web-infra-dev/rspack/issues/4124 they don't have this issue, have to figure out why.

Reply:
https://github.com/swc-project/swc/blob/4623b3e07f9cedb9652785f977f2bd0e509981d9/crates/swc_error_reporters/src/handler.rs#L53 swc's internal error can be controlled here